website/source/guides/identity/identity.html.md

```diff
@@ -0,0 +1,1123 @@
+- [Identity Secrets Engine (API)](/api/secret/identity/index.html)
+- [External vs Internal Groups](/docs/secrets/identity/index.html#external-vs-internal-groups)
+
+~> **NOTE:** An [interactive
+tutorial](https://www.katacoda.com/hashicorp/scenarios/vault-identity) is
+also available if you do not have a Vault environment to perform the steps
+described in this guide.
+
+## Estimated Time to Complete
+
+10 minutes
+
+## Personas
+
+The steps described in this guide are typically performed by **operations**
+persona.
+
+
+## Challenge
+
+Bob has accounts in both Github and LDAP.  Both Github and LDAP auth methods are
+enabled on the Vault server that he can authenticate using either one of his
+accounts. Although both accounts belong to Bob, there is no association between
+the two accounts to set some common properties.
+
+## Solution
+
+Create an _entity_ representing Bob, and associate aliases representing each of
+his accounts as the entity member. You can set additional policies and metadata
+on the entity level so that both accounts can inherit.
+
+When Bob authenticates using either one of his accounts, the entity identifier
+will be tied to the authenticated token. When such tokens are put to use, their
+entity identifiers are audit logged, marking a trail of actions performed by
+specific users.
+
+
+## Prerequisites
+
+To perform the tasks described in this guide, you need to have a Vault
+environment.  Refer to the [Getting
+Started](/intro/getting-started/install.html) guide to install Vault. Make sure
+that your Vault server has been [initialized and
+unsealed](/intro/getting-started/deploy.html).
+
+
+### Policy requirements
+
+-> **NOTE:** For the purpose of this guide, you can use the **`root`** token to work
+with Vault. However, it is recommended that root tokens are used for just
+enough initial setup or in emergencies. As a best practice, use tokens with
+an appropriate set of policies based on your role in the organization.
+
+To perform all tasks demonstrated in this guide, your policy must include the
+following permissions:
+
+```shell
+# Configure auth methods
+path "sys/auth" {
+  capabilities = [ "read", "list" ]
+}
+
+# Configure auth methods
+path "sys/auth/*" {
+  capabilities = [ "create", "update", "read", "delete", "list", "sudo" ]
+}
+
+# Manage userpass auth methods
+path "auth/userpass/*" {
+  capabilities = [ "create", "read", "update", "delete" ]
+}
+
+# Manage github auth methods
+path "auth/github/*" {
+  capabilities = [ "create", "read", "update", "delete" ]
+}
+
+# Display the Policies tab in UI
+path "sys/policies" {
+  capabilities = [ "read", "list", "sudo" ]
+}
+
+# Create and manage ACL policies from UI
+path "sys/policies/acl/*" {
+  capabilities = [ "create", "read", "update", "delete", "list", "sudo" ]
+}
+
+# Create and manage policies
+path "sys/policy" {
+  capabilities = [ "read", "list", "sudo"]
+}
+
+# Create and manage policies
+path "sys/policy/*" {
```

Comment:
`sys/policy/` is the older prefix, and I believe it's deprecated - since introducing sentinel policies there's `sys/policies/acl/*`

Reply:
Oh! maybe `vault policy write` still uses the old path - so may still be necessary. Sorry about that!